rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -160,10 +161,27 @@ error_code cellHddGameCheck(ppu_thread& ppu, u32 version, vm::cptr<char> dirName
 		return CELL_HDDGAME_ERROR_PARAM;
 	}
 
-	std::string dir = dirName.get_ptr();
+	std::string game_dir = dirName.get_ptr();
 
 	// TODO: Find error code
-	verify(HERE), dir.size() == 9;
+	verify(HERE), game_dir.size() == 9;
```

Comment:
Maybe remove this assert, if non HG content is accessed ERROR_BROKEN should be returned.

Reply:
This function can also create new directories, so I don't see the relevance of HG content in this topic.